src/index.ts

```diff
@@ -1,3 +1,4 @@
+require("isomorphic-fetch")
```

Comment:
Why is this needed? Any polyfill should be added to apps/tests harnesses using this library. Polyfills should not be required by libraries.